src/ConvertInputFormat/main.cpp

```diff
@@ -497,11 +496,10 @@ int main(int argc, const char *argv[])
 #endif
 
 #ifdef _OPENMP
-#pragma omp parallel default(none)                                                                                                                   \
-    shared(files, number_files, fileCount, schema, outputType, outputTypeStr, output_directory)
+#pragma omp parallel default(none) shared(files, number_files, fileCount, schema, outputType, outputTypeStr, output_directory)
     {
 #pragma omp for
-        for (std::size_t i = 0; i < number_files; ++i) {
+        for (int i = 0; i < number_files; ++i) {
```

Comment:
oops, didn't mean to commit this.  I had to do this to get it to compile in VS2017